src/components/chip-set/chip-set.tsx

```diff
@@ -295,6 +304,8 @@ export class ChipSet {
                     'force-invalid': this.isInvalid(),
                 }}
                 onClick={this.handleTextFieldFocus}
+                onMouseEnter={this.onMouseEnter}
+                onMouseLeave={this.onMouseLeave}
```

Comment:
I don't think we need these :slightly_smiling_face: 
We should be able to hide the button with only CSS rules and not have to rely on any JS code